nengo/synapses.py

```diff
@@ -356,6 +358,19 @@ def __call__(self, t, signal):
         def check(cls, A, B, C, D, X):
             return super().check(A, B, C, D, X) and (len(A) == 1 and (D == 0).all())
 
+    class OneXOneDim(OneX):
+        """ Step for systems with one state element, no passthrough, and a size-1 input.
```

Comment:
First line of a docstring shouldn't have a leading space, and there should be a blank line before the next paragraph (and should be left-aligned with the quotes)

Reply:
Done already in my WIP fixups